<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIRMenu</class>
 <widget class="QWidget" name="SIRMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>360</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>400</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>358</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_initial_values">
         <property name="title">
          <string>Society values</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_i">
              <property name="text">
               <string>Initial I</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_i">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSlider" name="slider_i">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_total">
              <property name="text">
               <string>Total society</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBox_total">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>2000</number>
              </property>
              <property name="value">
               <number>1000</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QSlider" name="slider_total">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Rates</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="2">
             <widget class="QSlider" name="slider_i_r">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_r_r">
              <property name="text">
               <string>Recovery rate</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_v_r">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>0.300000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.001000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_d_r">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>0.200000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.030000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSlider" name="slider_r_r">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_d_r">
              <property name="text">
               <string>Death rate</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSlider" name="slider_v_r">
              <property name="maximum">
               <number>300</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QSlider" name="slider_d_r">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_i_r">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>0.000100000000000</double>
              </property>
              <property name="maximum">
               <double>0.003000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.000100000000000</double>
              </property>
              <property name="value">
               <double>0.000500000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_r_r">
              <property name="minimum">
               <double>0.050000000000000</double>
              </property>
              <property name="maximum">
               <double>0.200000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.050000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_v_r">
              <property name="text">
               <string>Vaccination rate</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_i_r">
              <property name="text">
               <string>Infection rate</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_not_immune">
              <property name="text">
               <string>Not immune rate</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_not_immune">
              <property name="maximum">
               <double>0.400000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.020000000000000</double>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QSlider" name="slider_not_immune">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Time values</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="2">
           <widget class="QSlider" name="slider_total_time">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox_total_time">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>110</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Number of days</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>363</width>
        <height>129</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Results</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The number of infected people at the disease's peak.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Maximum number of simultaneous infected:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_max_I">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The number of days and hours that have passed since the beginning of simulation until there were no infected people.&lt;span style=&quot; font-weight:600;&quot;&gt; If not defined, the disease wasn't over in specified number of days.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Disease duration:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_duration">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of people who haven't been infected after the disease ended. (Not accurate for &amp;quot;Not immune rate&amp;quot; greater than 0).&lt;span style=&quot; font-weight:600;&quot;&gt; If not defined the disease wasn't over in specified number of days.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Not infected after disease:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_susceptible_and_vaccinated_after">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of people alive after the disease ended.&lt;span style=&quot; font-weight:600;&quot;&gt; If not defined the disease wasn't over in specified number of days.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Alive after disease</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_alive_after">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
